pallets/did/src/errors.rs

```diff
@@ -22,54 +22,54 @@ use crate::did_details::DidVerificationKeyRelationship;
 
 /// All the errors that can be generated when validating a DID operation.
 #[derive(Debug, Eq, PartialEq, TypeInfo)]
-pub enum DidError {
+pub enum Error {
 	/// See [StorageError].
-	StorageError(StorageError),
+	Storage(Storage),
 	/// See [SignatureError].
-	SignatureError(SignatureError),
+	Signature(Signature),
 	/// See [InputError].
-	InputError(InputError),
+	Input(Input),
 	/// An error that is not supposed to take place, yet it happened.
-	InternalError,
+	Internal,
 }
 
-impl From<StorageError> for DidError {
-	fn from(err: StorageError) -> Self {
-		DidError::StorageError(err)
+impl From<Storage> for Error {
+	fn from(err: Storage) -> Self {
+		Error::Storage(err)
 	}
 }
 
-impl From<InputError> for DidError {
-	fn from(err: InputError) -> Self {
-		DidError::InputError(err)
+impl From<Input> for Error {
+	fn from(err: Input) -> Self {
+		Error::Input(err)
 	}
 }
 
 /// Error involving the pallet's storage.
 #[derive(Debug, Eq, PartialEq, TypeInfo)]
-pub enum StorageError {
+pub enum Storage {
```

Comment:
What's the reason to remove the `Error` suffix here? If I see an enum called `Storage` I'm not expected error codes.

Reply:
Its part of the errors module, so its qualified name is did::errors::Storage which is beautiful. With the suffix it would be did::errors::StorageError.

Reply:
`did::errors::StorageError` would only be used once to bring `StorageError` into scope. After that it's always only `StorageError`. So your proposal is `errors::Storage` vs `StorageError`. I personally think `StorageError` is better since you can't forget to have the error path. If you don't use `errors::Storage` but `Storage` this becomes very misleading.